backend/overlapping-block-marker/mysqldb.py

```diff
@@ -0,0 +1,68 @@
+import SimpleHTTPServer
```

Comment:
instead of making a copy of this class, use the one in the simple haskell marker directory
```
sys.path.append(path to backend / simple haskell marker)
import mysqldb
```
this way you have the same management of passwords and environment variables